### DISCUSSION TITLE
How do you determine the dollar amount of a token?

### DISCUSSION THREAD

**Discussion by egnvk** [phantom/docs#315] [Help & Support] [answered]
### Summary

I mean for a large number of tokens how do you determine the value of the asset in dollars

### Steps to Reproduce

I mean for a large number of tokens how do you determine the value of the asset in dollars

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
@egnvk We use a combination of different price feed services.

**Reply by egnvk** [reply]
could you name these services? I do similar functionality. thanks

**Reply by adamdelphantom** [reply]
Currently birdeye and coin gecko.
